static/js/apps/nl_interface/debug_info.tsx

```diff
@@ -162,11 +162,12 @@ export interface DebugInfoProps {
 }
 
 export function DebugInfo(props: DebugInfoProps): JSX.Element {
-  const [showDebug, setShowDebug] = useState(false);
-
-  if (_.isEmpty(props.debugData)) {
+  const hideDebug =
+    document.getElementById("metadata").dataset.hideDebug === "True";
```

Comment:
sorry, is it quick to add a dbg= param to show it regardless?

that will be *very* helpful!